contrib/drivers/lvm/metrics_cli.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Please update copyright to `2019 OpenSDS Authors`.

Reply:
fixed